### ISSUE TITLE
Gateway network not set

### ISSUE THREAD

**Issue by jpmeijers** [ttnmapper/ttnmapper-android-v3#31]
hi, i may have a related problem, my measurements logged with the android app on 17/12/2021 show up in advanced maps (gw-id: cassiopeia-lps8, and also using my dev-id), but not on the main map yet. my gateway shows up.

_Originally posted by @tomtobback in https://github.com/ttnmapper/ttnmapper-web/issues/14#issuecomment-997613758_

**Comment by jpmeijers** [comment]
It looks like the antenna ID for the uplink message is not set.

@tomtobback can you confirm which version of the app you are using?

**Comment by tomtobback** [comment]
@jpmeijers i used the Android app version 2021.01.31 (build 31) as installed from Google Play on 17 Dec. as node i'm using an ESP32 with an SX1278 module, running an example of the arduino-lmic library. my gateway is a Dragino LPS8 running Basic Station -- TTN, pointing to the au1.cloud.thethings.network network server. i don't see any antenna_ID in the message json, please advise what i should check, thanks!

**Comment by tomtobback** [comment]
hi @jpmeijers - it seems the 'Advanced maps'  no longer shows any measurement data for my gateway 'cassiopeia-lps8'. i can do my measurements again but i first wanted to check with you if there's anything in my setup above that would again create the initial problem. you mentioned the 'antenna ID' but i'm not sure where to look for it. thanks

**Comment by jpmeijers** [comment]
@tomtobback there are two reasons for your gateways and data not appearing on the map. The root cause is still that the network ID is not set.

Can you please update to version 34 (2021.12.17) of the app? It seems like there is a bug in older versions of the app that caused the data uploaded from the app not to have the network ID set. This was fixed at the same time in build 34.

If the network ID of the gateway is not set, we do not know which network it belongs to, and the website do not know which icons to use for it. That's why the map is empty.

So in short, if you update to the latest version of the Android app, all should work from then on.

**Comment by tomtobback** [comment]
Hi @jpmeijers thanks for the hint. This morning's measurements are already visible on the main map now, all good, thanks a lot. One thing i notice is that on a city-wide scale, its heatmap shows, then zooming in it disappears until zoomed in to a local scale, maybe that's just because my furthest measurement was around 500m.

**Comment by jpmeijers** [comment]
Yes what you are seeing is likely caching of the tiles. Different zoom levels are cached for different periods. After max a day you should see your data on all zoom levels.
